ifrs17/Import/Importers.ipynb

```diff
@@ -622,6 +648,70 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Yield Curves "
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "Import.DefineFormat(ImportFormats.YieldCurve, async (options, dataSet) => {",
+                "\n    Activity.Start();",
+                "\n    var args = await GetArgsFromMainAsync(dataSet);",
+                "\n    if(ApplicationMessage.HasErrors())  return Activity.Finish();",
+                "\n",
+                "\n    var workspace = Workspace.CreateNew();",
+                "\n    workspace.Initialize(x => x.FromSource(DataSource).DisableInitialization<RawVariable>().DisableInitialization<IfrsVariable>());",
+                "\n    var oldYieldCurves = await DataSource.Query<YieldCurve>().ToArrayAsync();",
+                "\n",
+                "\n    var importLog = await Import.FromDataSet(dataSet)",
+                "\n        .WithType<YieldCurve>((dataset, datarow) => {",
+                "\n            var currency = datarow.Field<string>(nameof(YieldCurve.Currency));",
+                "\n            var values = datarow.Table.Columns.Where(c => c.ColumnName.StartsWith(nameof(YieldCurve.Values))).OrderBy(c => c.ColumnName.Length).ThenBy(c => c.ColumnName)",
+                "\n                                .Select(x => datarow.Field<string>(x.ColumnName).CheckStringForExponentialAndConvertToDouble()).ToArray();",
+                "\n            var item = new YieldCurve {",
+                "\n                Currency = currency,",
+                "\n                Year = args.Year,",
+                "\n                Month = args.Month, ",
+                "\n                Scenario = args.Scenario,",
+                "\n                Values = values",
+                "\n            };",
+                "\n            return item;",
+                "\n        }).WithTarget(workspace).ExecuteAsync(); ",
+                "\n    ",
+                "\n    if(importLog.Errors.Any()) return Activity.Finish().Merge(importLog); ",
+                "\n    ",
+                "\n    var newYieldCurves = await workspace.Query<YieldCurve>().ToArrayAsync();",
+                "\n    newYieldCurves = newYieldCurves.Except(oldYieldCurves, YieldCurveComparer.Instance()).ToArray();",
+                "\n    if (!newYieldCurves.Any()){",
+                "\n        ApplicationMessage.Log(Warning.VariablesAlreadyImported);   ",
```

Comment:
Can we add to the log some additional information such as which variable has already been imported?